src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureTableStorage.cs

```diff
@@ -44,16 +45,18 @@ public class AzureTableGrainStorage : IGrainStorage, IRestExceptionDecoder, ILif
 
         private const string BINARY_DATA_PROPERTY_NAME = "Data";
         private const string STRING_DATA_PROPERTY_NAME = "StringData";
-
+        private string name;
         /// <summary> Default constructor </summary>
-        public AzureTableGrainStorage(AzureTableStorageOptions options, SerializationManager serializationManager, IGrainFactory grainFactory, ITypeResolver typeResolver, ILoggerFactory loggerFactory)
+        public AzureTableGrainStorage(AzureTableStorageOptions options, SerializationManager serializationManager, IGrainFactory grainFactory, ITypeResolver typeResolver, ILoggerFactory loggerFactory, string name)
```

Comment:
NIT:  I think the name should be the first parameter, it's more clear
NIT²: Could you let a space between the latest member and the ctor? You are doing the same in the validator below
NIT³: When there is a lot of param like here, I think it is better to break them in multiple lines. It's more readable, and avoid merge conflicts sometimes